src/util.js

```diff
@@ -43,6 +43,10 @@ export const recursiveMerge = (a, b) => {
     return ret;
 };
 
+export const uniquify = (values) => {
+    return [...new Set(values)];
```

Comment:
I'm guessing this was already the plan, but I'd rather just not use `Set` than change this to include the polyfill (in the interest of keeping byte size down).